src/handlers.js

```diff
@@ -120,9 +120,14 @@ Handlers.prototype.completed = function(message) {
 
   // Get task definition
   return this.queue.task(message.payload.status.taskId).then(function(task) {
-    // Create default expiration date
-    var expires = new Date();
-    expires.setDate(expires.getDate() + 365);
+    
+    // Create expiration date
+    var expires = new Date();    
+    if(typeof(task.expires) !== 'undefined') {
+      expires.setDate(task.expires);
```

Comment:
Wouldn't it be easier to make an assignment, in both cases...
I'm just thinking what if setDate does something unexpected... Like not set the timestamp...